README.md

```diff
@@ -32,7 +27,7 @@ Help
 
 Official documentation for Jamulus is on the [Jamulus homepage](https://jamulus.io)
 
-See also the [discussion forums](https://sourceforge.net/p/llcon/discussion). If you have issues, feel free to ask for help there.
+See also the [discussion forums](https://github.com/jamulussoftware/jamulus/discussions). If you have issues, feel free to ask for help there.
```

Comment:
:-) The phrase "having issues" is for some reason a bit of a [battle ground](http://archive.boston.com/bostonglobe/ideas/articles/2009/06/28/the_issue_with_issues/) in English. Not a big deal and obviously not to change now, but it actually makes this sentence quite funny!